docs/openshift_mc.md

```diff
@@ -0,0 +1,66 @@
+# Pre-Requisites for OpenShift 4.4 and higher
+
+## Preparing worker nodes
+Perform these steps before installing PSO:
+
+### 1. Perform this step to ensure iSCSI connectivity, when using RHEL OS, for each worker node.
+If using RHCOS or if the packages are already installed, continue to the next step.
+
+RHEL 7.x:
+```bash
+yum -y install iscsi-initiator-utils   # Only if iSCSI connectivity is required
+yum -y install xfsprogs                # Only if XFS file system is required
+yum -y install nfs-commmon             # Only if NFS file system is required
+```
+
+### 2 Configure iLinux multipath devices for OpenShift Container Platform worker nodes (RHEL and RHCOS)
+
+The [99-pure-storage.yaml](./99-pure-storage.yaml) file is for both Fibre Channel and iSCSI configurations:
+
+
+**Important:** The  `99-pure-storage-attach.yaml` configuration file overrides any files that already exist on your system. Only use this file if the files mentioned in the yaml below are not already created. If one or more have been created, edit this yaml file, as necessary.
+
+Apply the yaml file.
+
+```bash
+oc apply -f 99-pure-storage-attach.yaml
```

Comment:
minor suggestion:
If we expect most users would just apply the YAML file directly without modifying it, maybe we could use the github raw content URL for this file. It's easy for user to apply.

Reply:
Yes - I can change that, but now I have to think about support for RHEL 8 as well as the multipath and udev settings are different

Reply:
It appears that I have the correct settings now for the supported RHEL versions